### DISCUSSION TITLE
Is there a chat to get help with this project?

### DISCUSSION THREAD

**Discussion by amckinlay** [SpenceKonde/ATTinyCore#670] [Support / Q & A]
Would be nice to be able to get help faster

**Comment by SpenceKonde** [comment]
No, there is no chat. I do not see that you have reported any problems or asked any questions....

**Reply by SpenceKonde** [reply]
We used Gitter in the past, but it was just like a forum that had less reliable notifications, because the chance of someone else who can help being on was too low.Never got enough uptake to be useful.
